bundles/core/org.eclipse.smarthome.core.thing.test/src/test/java/org/eclipse/smarthome/core/thing/internal/ChannelStateDescriptionProviderOSGiTest.java

```diff
@@ -153,22 +193,38 @@ public void presentItemStateDescription() {
                 null, "test thing", new Configuration());
         Assert.assertNotNull(thing);
```

Comment:
It would be nice to have all `assert*` methods to be static imports like in the other java tests.